tests/src/test/scala/scalaz/ValidationTest.scala

```diff
@@ -77,12 +78,12 @@ object ValidationTest extends SpecLite {
     "return first success" in {
       val succ = Validation.success[String, Int](1).toValidationNel
       val fail = Validation.failure[String, Int]("2").toValidationNel
-    
```

Comment:
After a small fight, I quite liked the way the `checkGenTypeClasses`? failed the build if the checkins didn't have classes matching the generated versions. This enforces that the author is responsible for the code when git blamed rather than an autoformatter/generator user. Of course, it doesn't solve the "how to fix the currently checked in code" issue.